project-base/app/src/DataFixtures/Demo/OrderDataFixture.php

```diff
@@ -863,6 +802,9 @@ private function createOrder(
             null,
         );
 
+        $uniqueOrderHash .= $orderData->firstName . $orderData->lastName . $orderData->transport->getId() . $orderData->deliveryFirstName . $orderData->deliveryLastName;
+        $orderData->uuid = Uuid::uuid5(self::UUID_NAMESPACE, md5($uniqueOrderHash))->toString();
```

Comment:
this hash is heavily dependent on the data in the order so changing them would lead to a different UUID of the order, right? I am not sure whether this is a good idea, however, I am not able to come up with a better solution (I was thinking about the order number but that is generated later, right?)

Reply:
maybe a simple counter as in the article data fixture would do the trick? In such a case, the UUIDs are sensitive to the ordering of the data fixtures but that seems to me to be less probable scenario than changing the order data here :thinking: